include/staff/templates/users.tmpl.php

```diff
@@ -34,9 +34,9 @@
 $page=($_GET['p'] && is_numeric($_GET['p']))?$_GET['p']:1;
 $pageNav=new Pagenate($total,$page,PAGE_LIMIT);
 $qstr = '&amp;'. Http::build_query($qs);
-$qs += array('sort' => $_REQUEST['sort'], 'order' => $_REQUEST['order']);
+$qs += array('id' => $org->getId(), 'sort' => $_REQUEST['sort'], 'order' => $_REQUEST['order']);
 
-$pageNav->setURL('users.php', $qs);
+$pageNav->setURL(basename($_SERVER['PHP_SELF']), $qs);
```

Comment:
You could hardcode that, because this template is only used in `incluce/staff/org-view.inc.php`. But this looks cleaner.